### PR TITLE
Kazakh term 6 310716

### LINKED ISSUE #14166: Kazakhstan: 6th Convocation
## Legislature

Kazakhstan (Mazhilis) — 6th Convocation

## Source

http://www.parlam.kz/en/mazhilis/General/Deputats

## Type of Data

List of Members

## Notes

Ideally we could scrape this in multiple languages.

### PR DESCRIPTION
**Pull request by ondenman**
Add data for term 6. (Second attempt)
Source: http://www.parlam.kz
Scraper: https://morph.io/everypolitician-scrapers/kazakhstan-mazhilis

Fixes #14166

### CODE REVIEW

File: data/Kazakhstan/Assembly/sources/instructions.json [hunk 1]
```diff
@@ -23,6 +23,33 @@
...
+      "type": "term"
+    },
+    {
+      "file": "gender-balance/results.csv",
+      "type": "gender",
+      "create": {
+        "from": "gender-balance",
+        "source": "Kazakhstan/Assembly"
+      }
+    },
```
Comment (tmtmtmtm) [inline data/Kazakhstan/Assembly/sources/instructions.json]: You seem to have doubled up the import of the terms file and the gender-balance file here…

File: data/Kazakhstan/Assembly/sources/manual/terms.csv [hunk 2]
```diff
@@ -1,2 +1,4 @@
 id,name,start_date,end_date,source
-5,Fifth Convocation,2012-01-15,,http://www.parlam.kz/en/mazhilis/history?id=history
+5,Fifth Convocation,2012-01-15,2016-01-20,http://www.ipu.org/parline-e/reports/2165_E.htm
+6,Fifth Convocation,2012-03-20,,http://www.ipu.org/parline-e/reports/2165_E.htm
```
Comment (tmtmtmtm) [inline data/Kazakhstan/Assembly/sources/manual/terms.csv]: This start date is highly suspicious, as it seems to be in the wrong year. Checking with the linked IPU page, I assume you _meant_ to set it to the date of the election (2016-03-20), but in general this should be set to the start date of the term (if known), which in this case is 2016-03-25.

**Comment by tmtmtmtm** [comment]
A few problems with double imports, and the wrong term date, but otherwise looks good!

**Comment by ondenman** [comment]
@tmtmtmtm Term dates corrected and duplicate gender balance instructions removed.

**Comment by tmtmtmtm** [comment]
@ondenman you're still importing the terms file twice, though…

**Comment by ondenman** [comment]
@tmtmtmtm Removed duplicate import instruction and regenerated data.

**Comment by ondenman** [comment]
@tmtmtmtm  squashed a8db54fd87b455607832adb47b40795cd0f2115a and 631ec7d181e464f9f191db0dbf7a9b4afc1cf46d

**Comment by tmtmtmtm** [comment]
I've squashed some of these commits down, and fixed up some leading whitespace you introduced, so this is good to go now!  👍

**Comment by tmtmtmtm** [comment]
Ooops, nope — not yet! From https://everypolitician-viewe-pr-13948.herokuapp.com/kazakhstan/ :

screen shot 2016-08-02 at 11 48 53

**Comment by tmtmtmtm** [comment]
You also need to add the Fixes/Closes for the original ticket (https://github.com/everypolitician/everypolitician-data/issues/14166)

**Comment by tmtmtmtm** [comment]
@ondenman You still need to rebuild the data after changing that, otherwise the Popolo is still wrong.

**Comment by tmtmtmtm** [comment]
Yay! 👍

**Pull request merged**